frontend/web/components/SegmentFeatureOverrides.js

```diff
@@ -344,7 +343,7 @@ class TheComponent extends Component {
                             />
                         </Flex>
                     )}
-                    {value && !!value.length && (
+                    {visibleValues && !!visibleValues.length && (
```

Comment:
Fixes a small issue where the UI showed segment priority dragging instructions where there were no overrides visible